content/en/blog/_posts/2022-11-04-live-and-let-live-with-kluctl-and-ssa.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: blog
+title: "Live and let live with Kluctl and Server Side Apply"
+date: 2022-11-04
+slug: live-and-let-live-with-kluctl-and-ssa
+---
+
+**Author:** Alexander Block (@codablock)
+
+This blog post was inspired by a previous Kubernetes blog post about
+[Advanced Server Side Apply](https://kubernetes.io/blog/2022/10/20/advanced-server-side-apply/).
+The author of said blog post listed multiple benefits for applications and
+controllers when switching to server-side apply (from now on abbreviated with
+SSA). Especially the chapter about
+[CI/CD systems](https://kubernetes.io/blog/2022/10/20/advanced-server-side-apply/#ci-cd-systems)
+motivated me to respond and write down my thoughts and experiences.
+
+These thoughts and experiences are the results of me working on [Kluctl](https://kluctl.io)
+for the past 2 years. I describe Kluctl as "The missing glue to put together
+large Kubernetes deployments, composed of multiple smaller parts
+(Helm/Kustomize/...) in a manageable and unified way."
+
+To get a basic understanding of Kluctl, I suggest to visit the [kluctl.io](https://kluctlio)
+website and read through the documentation and tutorials, for example the 
+[microservices demo tutorial](https://kluctl.io/docs/guides/tutorials/microservices-demo/).
+As an alternative, you can watch [Hands-on Introduction to kluctl](https://www.youtube.com/watch?v=9LoYLjDjOdg)
+from the Rawkode Academy YouTube channel which shows a hands-on demo session.
+
+There is also a [Kluctl delivery scenario](https://github.com/codablock/podtato-head/tree/kluctl/delivery/kluctl)
+available in the [podtato-head](https://github.com/codablock/podtato-head) demo project.
```

Comment:
```suggestion
available in my fork of the [podtato-head](https://github.com/codablock/podtato-head) demo project.
```
(it's OK to edit this line after the merge happens, even if the article has already been published).